.github/workflows/video-preview.yml

```diff
@@ -0,0 +1,21 @@
+name: Video
+on:
+  - pull_request
+jobs:
+  preview:
+    name: Preview
```

Comment:
Could you make this name be more meaningful? It'll show up on the left side in https://github.com/grahamegrieve/ClinicArrivals/actions.